### DISCUSSION TITLE
DeePMD incompatible with atom_style bond in LAMMPS?

### DISCUSSION THREAD

**Discussion by Sokseiha** [deepmodeling/deepmd-kit#1937] [Q&A]
Hi everyone,

I would like to add harmonic interaction to some bonds in addition to the the DeePMD forces. I managed to do this in LAMMPS using the atom_style bond and as a sanity check, I run the simulation with the spring constant k=0 and I expect the result to be the same as when I run the simulation with DeePMD alone. However, the results are quite different and I'm not sure why.
Is this an expected behavior or is there any (known) bugs/incompatibility in lammp-dp regarding this point?
I attach below the input files I used in case any of you want to reproduce this behavior. 
Thank you in advance for your help and please let me know if you have any questions.

Best,
[lammps_inputs.zip](https://github.com/deepmodeling/deepmd-kit/files/9615100/lammps_inputs.zip)

Seiha.

**Comment by wanghan-iapcm** [comment]
We need to do similar setting in DPLR: establishing virtual bonded pairs of atoms. Please let us know if the lammps configuration https://github.com/deepmodeling/deepmd-kit/blob/master/examples/water/dplr/lmp/in.lammps helps you
The bond style zero and setting of `special_bonds` for  the coefficients for pairwise energy and force contributions may be important 
```
bond_style	zero
bond_coeff	*
special_bonds   lj/coul 1 1 1 angle no
```
